### PR TITLE
Fix vendor issues

### PR DESCRIPTION
**Pull request by DylanVolz**

### CODE REVIEW

File: LICENSE [hunk 1]
```diff
@@ -321,12 +321,12 @@ For the jwt-go component:
 
 For the pq component:
 @traffic_ops/experimental/traffic_ops_auth/vendor/github.com/lib/pq/*
-@traffic_ops/testing/api/vendor/github.com/lib/pq/*
+@traffic_ops/vendor/github.com/lib/pq/*
 ./traffic_ops/experimental/traffic_ops_auth/vendor/github.com/lib/pq/LICENSE.md
```
Comment (rob05c) [inline LICENSE]: Looks like this line should be changed to `./traffic_ops/vendor/github.com/lib/pq/LICENSE.md` ?

File: LICENSE [hunk 2]
```diff
@@ -321,12 +321,12 @@ For the jwt-go component:
...
 
 and also:
-@traffic_ops/traffic_ops_golang/vendor/github.com/lib/pq/*
-./traffic_ops/traffic_ops_golang/vendor/github.com/lib/pq/LICENSE.md
+@traffic_ops/vendor/github.com/lib/pq/*
```
Comment (rob05c) [inline LICENSE]: Should this be removed, since it's the same line as above now?

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1505/
Test PASSed.

File: traffic_ops/build/traffic_ops.spec [hunk 3]
```diff
@@ -67,7 +67,7 @@ Built: %(date) by %{getenv: USER}
     # Create build area with proper gopath structure
     mkdir -p src pkg bin || { echo "Could not create directories in $(pwd): $!"; exit 1; }
 
-
+
```
Comment (rob05c) [inline traffic_ops/build/traffic_ops.spec]: Nitpick: trailing whitespace makes my editor swear.

File: LICENSE (hunk 1 above)
Comment (DylanVolz) [reply LICENSE]: well they all are referencing the license file at the bottom of the list I believe, so the 3 with @'s are all using the LICENSE.md under experimental.

File: LICENSE (hunk 2 above)
Comment (DylanVolz) [reply LICENSE]: correct. Will update the PR.

File: traffic_ops/build/traffic_ops.spec (hunk 3 above)
Comment (DylanVolz) [reply traffic_ops/build/traffic_ops.spec]: +1

**Comment by asfgit** [comment]
Refer to this link for build results (access rights to CI server needed): 
https://builds.apache.org/job/incubator-trafficcontrol-PR/1506/
Test PASSed.

**Pull request merged**